### DISCUSSION TITLE
编译报错，请问一下怎么解决

### DISCUSSION THREAD

**Discussion by wanansany** [coolsnowwolf/lede#12544] [Q&A / 快速问答]
Compiling build_helper v0.1.0 (/opt/lede/build_dir/target-aarch64_generic_musl/host/rustc-1.78.0-src/src/tools/build_helper)
    Finished dev [unoptimized] target(s) in 37.06s
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 121754` at the top of `config.toml`
downloading https://ci-artifacts.rust-lang.org/rustc-builds/9b00956e56009bab2aa15d7bff10916599e3d6d6/rust-dev-1.78.0-x86_64-unknown-linux-gnu.tar.xz
curl: (22) The requested URL returned error: 404                                                

ERROR: failed to download llvm from ci

    HELP: There could be two reasons behind this:
        1) The host triple is not supported for `download-ci-llvm`.
        2) Old builds get deleted after a certain time.
    HELP: In either case, disable `download-ci-llvm` in your config.toml:

    [llvm]
    download-ci-llvm = false
    
Build completed unsuccessfully in 0:01:03
make[3]: *** [Makefile:110: /opt/lede/build_dir/target-aarch64_generic_musl/host/rustc-1.78.0-src/.built] Error 1
make[3]: Leaving directory '/opt/lede/feeds/packages/lang/rust'
time: package/feeds/packages/rust/host-compile#156.87#39.44#97.06
    ERROR: package/feeds/packages/rust [host] failed to build.
make[2]: *** [package/Makefile:114: package/feeds/packages/rust/host/compile] Error 1
make[2]: Leaving directory '/opt/lede'
make[1]: *** [package/Makefile:110: /opt/lede/staging_dir/target-aarch64_generic_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/opt/lede'
make: *** [/opt/lede/include/toplevel.mk:231: world] Error 2

**Comment by SpeedPartner** [comment]
https://github.com/coolsnowwolf/packages/commit/741eb4376c1a78fd27c8a89ac8a0a14da0c527a9
此问题目前已修复
